backends/postgres/db.ts

```diff
@@ -146,6 +168,8 @@ function dbFor(useClient: UseClient): DB {
   };
 }
 
-await usePool(queryObject(schema)); // creating db schema.
+if (Deno.env.get("PGDATABASE")) {
+  await usePool(queryObject(schema)); // creating db schema.
+}
```

Comment:
I think it's OK to make the `postgre()` function be async and you can move this code to there

Reply:
You only need to check the env var once, when deciding which database we should use